libenkf/src/ecl_config.c

```diff
@@ -953,6 +920,11 @@ bool ecl_config_get_unified_summary(const ecl_config_type * ecl_config)
   return ecl_io_config_get_unified_summary(ecl_config->io_config);
 }
 
+bool ecl_config_have_eclbase(const ecl_config_type * ecl_config) {
+  return ecl_config->have_eclbase;
+}
+
+
 void ecl_config_add_config_items(config_parser_type * config)
 {
   config_schema_item_type * item;
```

Comment:
It we are not supporting to provide a target schedule file any more, please update the definition of SCHEDULE_FILE_KEY accordingly.